checkers/checkers.go

```diff
@@ -2,15 +2,8 @@
 package checkers
 
 import (
-	"fmt"
-	"go/ast"
-	"go/build"
-	"go/token"
-	"os"
-
-	"github.com/go-critic/go-critic/checkers/rulesdata"
 	"github.com/go-critic/go-critic/framework/linter"
-	"github.com/quasilyte/go-ruleguard/ruleguard"
+	"os"
```

Comment:
Nit: it would be better to move"os" before ""github.com/go-critic/go-critic/framework/linter" and keep new line between them ;)